pkg/shell/indexes.js

```diff
@@ -124,6 +125,66 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
             index.preload_frames(m, m.manifests);
     }
 
+    function selectActiveItem() {
+        const cur = document.activeElement;
+        if (cur.nodeName === "INPUT") {
+            const el = document.querySelector("#host-apps li:first-of-type > a");
+            if (el)
+                el.click();
+        } else {
+            cur.click();
+        }
```

Comment:
This function could use some documentation what it does. "select" also clashes with "click".